### PR TITLE
Update g-suite-apps-audit-source.md

### PR DESCRIPTION
**Pull request by kparekh010**
## Purpose of this pull request

This pull request (PR) is to have a single source of truth between the cse docs and send data docs. 

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md [hunk 1]
```diff
@@ -39,16 +39,16 @@ Authentication must be with a new G Suite Apps Audit Source, we do not support
 
 ## Configuring a G Suite Apps Audit Source
 
-Once you've set up a Hosted Collector and have your credentials ready, you're all set to configure the Source.
+Once you've set up a Hosted Collector and have your credentials ready, you're all set to configure the Source. Each G Suite App must have its own source. Follow these steps for each G Suite App you wish to collect.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md:42]:
```suggestion
Once you've set up a Hosted Collector and have your credentials ready, you're all set to configure the Source. Each G Suite App must have its own source. For each G Suite App you wish to collect, follow the steps below.
```

File: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md [hunk 2]
```diff
@@ -39,16 +39,16 @@ Authentication must be with a new G Suite Apps Audit Source, we do not support
...
 
 To configure a G Suite Apps Audit Source:
 
 1. In Sumo Logic select **Manage Data \> Collection > Collection**.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md:46]:
```suggestion
1. In Sumo Logic select **Manage Data** > **Collection** > **Collection**. 
```

File: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md [hunk 3]
```diff
@@ -39,16 +39,16 @@ Authentication must be with a new G Suite Apps Audit Source, we do not support
...
 
 To configure a G Suite Apps Audit Source:
 
 1. In Sumo Logic select **Manage Data \> Collection > Collection**. 
 1. In the Collectors page, click **Add Source** next to a Hosted Collector.
 1. Select **G Suite Apps Audit**.
 1. For **Name**, type the name of the Source. Enter an optional Description.
-1. For **Application**, select the app that you’d like this Source to collect data from.
-1. Type the **Source Category** you’d like to use to later search the data collected from this Source.
+1. For **Application**, Select the G Suite app you wish to collect using this source. Steps may be repeated for each G Suite app you want to collect from.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md:50]:
```suggestion
1. For **Application**, select the G Suite app that you’d like this Source to collect data from. Steps may be repeated for each G Suite app you want to collect from.
```

File: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md [hunk 4]
```diff
@@ -39,16 +39,16 @@ Authentication must be with a new G Suite Apps Audit Source, we do not support
...
+1. **Source Category**. Enter a string to tag the output collected from the source. The string that you supply will be saved in a metadata field called `_sourceCategory.`
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md:51]:
```suggestion
1. **Source Category**. Enter a string to tag the output collected from the source. The string that you supply will be saved in a metadata field called `_sourceCategory`.
```

**Review by JainM6 — DISMISSED** [review]
Minor formatting

**Review by kimsauce — APPROVED** [review]

**Pull request merged**